src/main/java/duke/command/AddCommand.java

```diff
@@ -0,0 +1,62 @@
+package duke.command;
+
+import duke.error.IncorrectFormat;
+import duke.parts.Storage;
+import duke.parts.TaskList;
+import duke.parts.Ui;
+import duke.task.Deadline;
+import duke.task.Task;
+import duke.task.ToDo;
+import duke.task.Event;
+
+/**
+ * Represents a command which is used to add a task to the list.
+ * It is executed when the execute method is called.
+ */
+public class AddCommand extends Command{
+    String input;
+
+    public AddCommand(String input) {
+        this.input = input;
+    }
+
+    /**
+     * Executes the add command.
+     * It will create and add the task as specified by the input when the command is created.
+     * @param tasks The task list of the system.
+     * @param ui The UI of the system which interacts with user.
+     * @param storage The storage of the system.
+     * @throws IncorrectFormat
+     */
+    @Override
+    public void execute(TaskList tasks, Ui ui, Storage storage) throws IncorrectFormat {
+        int space = input.indexOf(" ");
+        int slash = input.indexOf("/");
```

Comment:
Would it be better if you name these 2 variables like indexOfSpace/Slash or positionOfSpace/Slash? Although I can understand what you mean, and it causes no confusion in this context, the meaning of the variables seem a bit ambiguous.